packages/eckit/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+from spack.pkg.builtin.eckit import Eckit as SpackEckit
+
+
+class Eckit(SpackEckit):
+
+    git = 'https://github.com/ecmwf/eckit.git'
+    version('1.20.0', branch='1.20.0')
+
+    def flag_handler(self, name, flags):
```

Comment:
Should we create a PR with this enhancement also in spack? Or create a GitHub issue in spack-c2sm?